PPSTools/Utilities/interface/PPSOpticsCalibrator.h

```diff
@@ -0,0 +1,94 @@
+#ifndef PPS_OPTICS_CALIBRATION
+#define PPS_OPTICS_CALIBRATION
+#include "PPSTools/Utilities/interface/PPSUtilities.h"
+#include <boost/program_options.hpp>
+#include "TROOT.h"
+#include "Rtypes.h"
+#include "TH2F.h"
+#include "TTree.h"
+#include "TDirectory.h"
+#include "TSystem.h"
+#include "TRandom3.h"
+#include "TMath.h"
+#include "TLorentzVector.h"
+#include "TString.h"
+#include "TFile.h"
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <cmath>
+#include <cstdio>
+#include "H_BeamParticle.h"
+#include "H_BeamLine.h"
+#include <tuple>
+#include <string>       // std::string
+#include <iostream>     // std::cout
+#include <sstream>      // std::stringstream, std::stringbuf
+
+static const double cm_to_m=0.1;
```

Comment:
seems this is duplicated even in this PR (let alone beyond it)

Reply:
deleted... using the instance in PPSUnitConversion.h instead.